packages/dataviews/src/dataviews-layouts/list/style.scss

```diff
@@ -34,6 +34,7 @@ ul.dataviews-view-list {
 				flex-basis: min-content;
 				overflow: unset;
 				margin-inline: $grid-unit-10 0;
```

Comment:
It seems from quick testing that this `margin` is no longer relevant and can be removed if you’d like.